WordPress/Classes/Services/ReaderSearchSuggestionService.swift

```diff
@@ -0,0 +1,83 @@
+import Foundation
+
+///
+///
```

Comment:
Missing comment perhaps? (please, check the whole file, seeing many empty `//`'s